packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -224,14 +249,24 @@ export async function registerCheckpoints (
       }
       case 'appl': {
         txConfirmation = await deployer.waitForConfirmation(txn.txID());
-        const sscInfo: rtypes.SSCInfo = {
-          creator: encodeAddress(txn.from.publicKey),
-          txId: txn.txID(),
-          appID: txConfirmation['application-index'],
-          confirmedRound: txConfirmation['confirmed-round'],
-          timestamp: Math.round(+new Date() / 1000)
-        };
+        const key = deployer.getAppCheckpointKeyFromIndex(txn.appIndex);
+        if (key) {
+          const temp: rtypes.SSCInfo | undefined = deployer.getSSCfromCPKey(key);
+          if (txn.appOnComplete === 5 && temp) {
```

Comment:
instead of 5 can we use `TxnOnComplete.DeleteApplication` (taken from types-algosdk)?

Reply:
yes

Reply:
![asdasdas](https://user-images.githubusercontent.com/26249903/121872067-77f41880-cd22-11eb-9279-72126e8e3466.png)

Reply:
the code seems to be showing that, or we can add a comment with a link, let's add a object